cmake/find/FindNASM.cmake

```diff
@@ -0,0 +1,35 @@
+# This module will set `NASM_FOUND` if NASM is found.
+# If NASM is found, `NASM_ROOT` will be set to the root folder.
+# `NASM_BIN_DIR` will be set to the folder containing the executable,
+# and `NASM_EXECUTABLE` will be set to the path of the NASM executable.
+
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindNASM module")
+endif()
+
+# Try to find it even if it may already be set.
+find_path(
```

Comment:
Generally, we do not run nasm by ourself but by adding the bin folder to `PATH` and let the library building script find it. So actually, the executable is rather useless.